Vagrantfile

```diff
@@ -0,0 +1,48 @@
+# Specify minimum Vagrant version and Vagrant API version
+Vagrant.require_version ">= 1.6.0"
+VAGRANTFILE_API_VERSION = "2"
+
+# Create box
+Vagrant.configure("2") do |config|
+  config.vm.define "docker-lvm-plugin-fedora32"
```

Comment:
Can you update this to fedora 33. Fedora 32 will be out of date in June.

Reply:
Updated to fedora 33.